Makefile

```diff
@@ -74,14 +74,14 @@ $(BINARIES): bin/%:
 mod-tidy:
 	$(Q)for mod in $$(find . -name go.mod); do \
 	    echo "Tidying $$mod..."; ( \
-	        cd $$(dirname $$mod) && go mod tidy \
+	        cd $$(dirname $$mod) && go mod tidy -go=$$(grep -E "go\s+\d\.\d+" go.mod | sed 's/go //') \
```

Comment:
nit: maybe it's just me but my gut feeling would like 'go ' pattern to be forced to only match at the beginning of the line  with "^go...".